src/Serilog/Formatting/Display/LogEventLevelValue.cs

```diff
@@ -0,0 +1,116 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Serilog.Events;
+using Serilog.Parsing;
+
+namespace Serilog.Formatting.Display
+{
+    // allows for the specific handling of the {Level} element.
+    // can now have a fixed width applied to it, as well as casing rules.
+    internal class LogEventLevelValue : LogEventPropertyValue
+    {
+        private readonly LogEventLevel _value;
+
+        // Assumes reading from Dictionary<,> is thread-safe after construction
+        private static readonly Dictionary<LogEventLevel, string[]> _shortenedLevelMap
+            = new Dictionary<LogEventLevel, string[]>
+                  {
+                      { LogEventLevel.Verbose, new []{ "V", "Vb", "Vrb", "Verb" } },
+                      { LogEventLevel.Debug, new []{ "D", "De", "Dbg", "Dbug" } },
+                      { LogEventLevel.Information, new []{ "I", "In", "Inf", "Info" } },
+                      { LogEventLevel.Warning, new []{ "W", "Wn", "Wrn", "Warn" } },
+                      { LogEventLevel.Error, new []{ "E", "Er", "Err", "Eror" } },
+                      { LogEventLevel.Fatal, new []{ "F", "Fa", "Ftl", "Fatl" } },
+                  };
+
+        public LogEventLevelValue(
+            LogEventLevel value)
+        {
+            _value = value;
+        }
+
+        /// <summary>
+        /// This method will apply only upper or lower case formatting, not fixed width
+        /// </summary>
+        public override void Render(
+            TextWriter output, 
+            string format = null, 
+            IFormatProvider formatProvider = null)
+        {
+            ApplyFormatting(output, _value.ToString(), null, format);
+        }
+
+        /// <summary>
+        /// Will apply fixed width rules using the provided alignment
+        /// </summary>
+        public void Render(
+            TextWriter output,
+            Alignment? alignment,
+            string format = null,
+            IFormatProvider formatProvider = null)
+        {
+            ApplyFormatting(output, AlignedValue(alignment), alignment, format);
+        }
+
+        private string AlignedValue(Alignment? alignment)
+        {
+            if (!alignment.HasValue || alignment.Value.Width <= 0)
+            {
+                return _value.ToString();
+            }
+
+            if (_value.ToString().Length == alignment.Value.Width)
```

Comment:
This `if`/block isn't needed, since the behaviour will be the same without it - if the value has a custom width we'll use that in the next block, if it's too wide we'll trim it in the one after, or finally, we'll do what this block does and return it as `ToString()`.